feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -0,0 +1,79 @@
+# RT-1.1: Base BGP Session Parameters
+
+## Summary
+
+BGP session establishment between DUT - ATE and verifiying different session parameters.
+
+## Topology
+
+*   DUT port-1 ------- ATE port-1    
+
+## Procedure
+
+*   Establish eBGP session between DUT and ATE 
+
+    *   Ensure session state should be Established.
+    *   Verify BGP capabilities like Route refresh, ASN32 and MPBGP.
+
+*   Verify BGP session disconnect by sending notification message from ATE
+
+    *   Send Cease notification from ATE. 
+    *   Ensure that DUT telemetry correctly reports the error code.
+
+*   Establish BGP session and verify different session parameters. 
+    BGP session should be established when different session parameters
+    are provided and session parameters are applied correspondingly.
+
+    *   Explicitly specified Router ID under bgp global level.
```

Comment:
Since it's been a while since I looked at this test plan revision, I now have some additional thoughts.

It's now apparent to me that most of the parameters are disjoint, we can probably simplify the cases into just three or four major ones:

* iBGP - global AS
* eBGP - global AS
* eBGP - neighbor AS (does global AS still need to be specified?)
* (do we need an iBGP case for neighbor AS?)

The other parameters such as router ID, MD5 auth, holdtime and retry interval can just be set in all those three cases.

Also, let's state in the test plan which local and remote AS numbers have been picked for these cases.

Reply:
@liulk Sure, I will work on these changes and submit changes.

Reply:
@liulk I have summarized below test steps to automate  for RT1.1. Please could you confirm if these test steps are good to continue.

1.) iBGP -  
              i.) Verify with Global AS, 
              ii.) Verify with neighbor AS 
              iii.) Verify Session disconnect , receipt of cease error code
              iv.) Verify all session parameters - Route ID, MD5 Auth, Hold timer and Conn Retry Timer

2.) eBGP - 
              i.) Verify with Global AS, 
              ii.) Verify with neighbor AS 
              iii.) Verify Session disconnect , receipt of cease error code
              iv.) Verify all session parameters - Route ID, MD5 Auth, Hold timer and Conn Retry Timer

Thanks,
Prabha

Reply:
Hi Prabha, I updated README.md to clarify the procedure (you might need to rebase). Please take a look and let me know if that is good to continue.

Reply:
@liulk , thank you, I am back from PTO.
I have checked README file, will update script 8/26 and submit it for review.